### PR TITLE
Add hoferlayfs and bilderrahmen integration

### PR DESCRIPTION
**Pull request by karras**
This PR adds currently missing core features.

### CODE REVIEW

**Comment by hairmare** [comment]
👀

**Comment by winged** [comment]
needs rebase :)

File: flappy.txt [hunk 1]
```diff
@@ -14,5 +14,10 @@ git push -f stands for friday
 i herd u liek fallblatt
 flappy sez need more flappy text
 i like to flap it flap it
+in the past people looked up to me
+You must pass XXX prefix to password
```
Comment (winged) [inline flappy.txt]: Good one! <3

**Comment by karras** [comment]
I've already executed `git pull` to squash the merge conflict commit but you were too fast. :)

**Pull request merged**